python/servo/post_build_commands.py

```diff
@@ -95,8 +95,18 @@ def run(self, params, release=False, dev=False, android=False, debug=False, debu
                 print("Could not find a suitable debugger in your PATH.")
                 return 1
 
+            command = self.debuggerInfo.path
+            if debugger == 'gdb' or debugger == 'lldb':
+                rustCommand = 'rust-' + debugger
+                try:
+                    result = subprocess.call([rustCommand, '--version'], env=env, stdout=open(os.devnull, 'w'))
```

Comment:
Sorry, the second part was silly. I wonder though why you suggested check_output() rather than just check_call()?

Anyway, the first part still holds. Though now I'm wondering whether check_call() (*without* `which`) wouldn't be slightly cleaner than call()...